src/main/java/dev/openfeature/sdk/providers/memory/InMemoryProvider.java

```diff
@@ -82,7 +82,7 @@ public void updateFlags(Map<String, Flag<?>> flags) {
     public void updateFlag(String flagKey, Flag<?> flag) {
         this.flags.put(flagKey, flag);
         ProviderEventDetails details = ProviderEventDetails.builder()
-            .flagsChanged(Arrays.asList(flagKey))
+            .flagsChanged(Collections.singletonList(flagKey))
```

Comment:
This change is to make it clear to the next developer that it is the intention to only have a single value passed into this parameter.

Reply:
I like it.